tools/cache_monitor.py

```
@@ -0,0 +1,441 @@
+            },
+            "target": {
+                "type": "string",
+                "description": (
+                    "Target for action: 'all' (all caches), 'token' (token cache), "
+                    "'schema' (schema cache), 'model' (model validation cache), "
+                    "or specific model name for invalidation"
+                ),
+            },
+        }
+
+    def prepare_prompt(self, request: CacheMonitorRequest) -> str:
```

Pylint (3.3.7)

[warning] Method 'prepare_prompt' was expected to be 'async', found it instead as 'non-async'

(W0236)